infrastructure/variables.tf

```diff
@@ -137,6 +137,24 @@ variable "elastic_search_case_index_type" {
   default = "_doc"
 }
 
+variable "elastic_search_nodes_discovery_enabled" {
+  description = "Enable Elasticsearch node discovery by Jest client"
+  type = "string"
+  default = "true"
+}
+
+variable "elastic_search_nodes_discovery_frequency_millis" {
+  description = "Elasticsearch node discovery frequency in milliseconds"
+  type = "string"
+  default = "1000"
+}
+
+variable "elastic_search_nodes_discovery_filter" {
+  description = "Elasticsearch node discovery filter"
+  type = "string"
+  default = "_all"
```

Comment:
defaulting discovery of nodes to all ES nodes. This can be filtered to specific node patterns e.g. master, data etc. As all our ES nodes are data nodes, just using the all for now

Reply:
ok

Reply:
we can easily restrict the discovery to data nodes only in that case